src/libraries/Microsoft.PowerFx.Core/Public/Types/FormulaType.cs

```diff
@@ -233,7 +236,12 @@ public override int GetHashCode()
         internal virtual void DefaultExpressionValue(StringBuilder sb)
         {
             throw new NotSupportedException($"{GetType().FullName} doesn't implement DefaultExpressionValue.");
-        }
+        }
+
+        internal virtual void DefaultUniqueExpressionValue(StringBuilder sb)
+        {
+            DefaultExpressionValue(sb);
+        }

```

Comment:
When is this necessary? Would we ever expect serializers to not use unique expression value?